crates/full-node/sov-sequencer/src/preferred/nonce_buffer_task.rs

```diff
@@ -601,25 +637,11 @@ impl<E: TxExecutionBackend<S, Rt> + Send + 'static, S: Spec, Rt: Runtime<S>>
             tracing::warn!("Sequencer nonce buffer input receiver dropped. Assuming sequencer shutdown. Error: {e:?}");
             SequencerStateUpdatorError::Shutdown
         })?;
-        let nonce_when_queued = self
-            .execution_backend
-            .get_current_nonce_for_user(&credential_id);
-        let queued_at = Instant::now();
         queue_receiver.await.unwrap_or_else(|_| {
-            // The oneshot sender was dropped. This should normally only happen
-            // on shutdown, or if there's a bug.
-            let current_nonce = self
-                .execution_backend
-                .get_current_nonce_for_user(&credential_id);
-            err_invalid_nonce::<S, Rt>(
-                tx_hash,
-                tx_nonce,
-                current_nonce,
-                nonce_when_queued,
-                queued_at,
-                credential_id,
-                InvalidNonceReason::EvictedBeforeExecution,
-            )
+            // The oneshot sender was dropped. This might happen on shutdown, but will also happen
```

Comment:
Should we explicitly send a response when we drop the channel rather than relying on implicit behavior? I'm worried that we might have other reasons to drop the channel, and this error will mask the actual behavior